Adds command `spo file version restore`, closes #3956 (#3967)
pnp/cli-microsoft365: docs/docs/cmd/spo/file/file-version-restore.md

# spo file version restore

Restores a specific version of a specified file

## Usage

```sh
m365 spo file version restore [options]
```

## Options

`-w, --webUrl <webUrl>`
: The URL of the site where the file is located

> `--label <label>`
> : Label of version which will be retrieved
> 
> `-u, --fileUrl [fileUrl]`
> : The server-relative URL of the file to retrieve. Specify either `fileUrl` or `fileId` but not both
> 
> `-i, --fileId [fileId]`
> : The UniqueId (GUID) of the file to retrieve. Specify either `fileUrl` or `fileId` but not both

Reviewer: ```suggestion
`--label <label>`
: Label of version which will be restored

`-u, --fileUrl [fileUrl]`
: The server-relative URL of the file whose version will be restored. Specify either `fileUrl` or `fileId` but not both

`-i, --fileId [fileId]`
: The UniqueId (GUID) of the file whose version will be restored. Specify either `fileUrl` or `fileId` but not both
```
